Makefile

```diff
@@ -10,7 +10,7 @@
 
 .PHONY: build
 build: build-deps
-	go build -mod=vendor -o pd ./command
+	go build -mod=vendor -o out/pd ./pd/main.go
```

Comment:
Since we ignore `vendor/` it doesn't seem like there's value in having `-mod=vendor` on this command. Since you're already editing it, may as well remove that.